apps/frontend/src/pages/no-access/no-access.tsx

```diff
@@ -0,0 +1,20 @@
+import { PageLayout } from "~/libs/components/components.js";
+
+import styles from "./styles.module.css";
+
+const NoAccess = (): JSX.Element => {
+	return (
+		<PageLayout>
+			<div className={styles["container"]}>
+				<h1 className={styles["title"]}>
+					You do not have permission to access this page.
+				</h1>
+				<p className={styles["text"]}>
+					Please contact an administrator to request access.
```

Comment:
let's slightly rename it. Also please set max-width and align text to the center, so it looks pretty and not too wide 
```suggestion
					Pending access
				</h1>
				<p className={styles["text"]}>
					You don't have access to any pages for now. Please contact an admin to enable access for you.
```